Meta/CMake/vcpkg/distribution-triplets/x64-windows-static.cmake

```diff
@@ -0,0 +1,3 @@
+# Note: This is equivalent to upstream x64-windows-static-md-release.cmake
```

Comment:
this file is named -static due to this logic in generate_toolchain_variables.cmake 

```cmake
if (NOT triplet_path STREQUAL "distribution")
      if (NOT os_name MATCHES "Windows") #NOTE: Windows defaults to dynamic linking
            set(full_triplet "${full_triplet}-dynamic")
        endif()
elseif (os_name MATCHES "Windows")
    set(full_triplet "${full_triplet}-static")
endif()
```

Reply:
actually this doesn't work. why doesn't it select `x64-windows-static` for VCPKG_TARGET_TRIPLET when using `-DVCPKG_OVERLAY_TRIPLETS="C:\Users\Andrew\source\repos\ladybird\Meta\CMake\vcpkg\distribution-triplets"`

Reply:
Fixed with proper cmake_path usage.